R/sits_kohonen.R

```diff
@@ -215,6 +218,11 @@ sits_cluster_som <-
 
 sits_clean_samples_som <- function(som_cluster_analysis.tb) {
 
+    sample.tb <- som_cluster_analysis.tb$samples.tb
```

Comment:
Why do you need `sample.tb` in these new lines? This variable is not used anywhere else in this function.

Reply:
This has been solved in version 0.9.1